Solutions/42Crunch API Protection/Analytic Rules/APIJWTValidation.yaml

```diff
@@ -0,0 +1,42 @@
+id: bbd163f4-1f56-434f-9c23-b06713c119c2
+name: API - JWT validation
+description: |
+  '42Crunch API protection against JWT validation'
+severity: Low
+requiredDataConnectors:
+  - connectorId: 42CrunchAPIProtection
+    dataTypes:
+      - apifirewall_log_1_CL
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+status: Available
+tactics:
+  - CredentialAccess
+relevantTechniques:
+query: |
+  apifirewall_log_1_CL 
+  | where TimeGenerated >= ago(5m) 
+  | where Error_Message_s contains "missing [\"x-access-token\"]" 
```

Comment:
This one may require `contains `versus changing to `has`, but please confirm via testing.